frontend/src/components/object/ObjectTable.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import { storeToRefs } from 'pinia';
+import { FilterMatchMode } from 'primevue/api';
```

Comment:
Couldn't find too much documentation on this. So was wondering is lazy loading supported? How the search works in general, for the visible records, only or?

Reply:
`FilterMatchMode` component should be re-wrapped in our `@/lib/primevue` module.

Reply:
@dc-bcgov yeah the primevue data search or filter is just looking through what is in the table. So API lookup based on input from that field is not supported in this cut of BCBox.
https://primevue.org/datatable/#filter

@jujaga fixed. Different export format for that API file